BUILDING.md

```diff
@@ -55,43 +60,32 @@ Either do this globally or in your shell every time.
 
 Make sure the environment is setup before you run bazel (`bazel shutdown` will shut it down).
 
-1. Make sure there is no instance of go on your `PATH`
-Run `where go`. If a version of go is found, please remove it from `PATH`.
+1. Add `C:\tools\msys64\mingw64` to the PATH:
+   `set PATH=C:\tools\msys64\mingw64\bin;%PATH%`
+   Running `where gcc` should now find mingw’s gcc.
 
-1. Add `c:\tools\msys64\mingw64` to the PATH 
-`set PATH=c:\tools\msys64\mingw64\bin;%PATH%`
+1. Add `C:\tools\python27` to the PATH:
+   `set PATH=C:\tools\python27;%PATH%`
+   Alternatively, pass the path to python via the `--python_path` to bazel. See the [bazel documentation](https://docs.bazel.build/versions/master/windows.html#build-python) for more info.
 
-Running `where gcc` should now find mingw’s gcc.
+1. Set TMP and TEMP to something very short. `C:\tmp` is known to work. For faster builds, add this folder to the excemptions of the Windows Defender anti-malware scanner.
 
 The following environment variables will need to be set prior to building:
 
-| Variable            | Target                            |
-| ------------------- | --------------------------------- |
-| `JAVA_HOME`         | Path to the Java Runtime          |
-| `ANDROID_HOME`      | Path to Android SDK               |
-| `ANDROID_NDK_HOME`  | Path to Android NDK               |
-| `BAZEL_SH`          | `C:\tools\msys64\usr\bin\bash.exe`|
-| `GOPATH`            | \<unset\>                         |
-| `GOROOT`            | \<unset\>                         |
-| `TMP`               | `c:\tmp`                          |
-
-
-### Building
-
-In a terminal type:
-```
-cd <path-to-gapid-source>
-bazel build --config mingw pkg
-```
-The build output will be at `bazel-bin/pkg`.
+| Variable            | Target                             |
+| ------------------- | ---------------------------------- |
+| `ANDROID_HOME`      | Path to Android SDK                |
+| `ANDROID_NDK_HOME`  | Path to Android NDK                |
+| `BAZEL_SH`          | `C:\tools\msys64\usr\bin\bash.exe` |
+| `TMP` and `TEMP`    | `C:\tmp`                           |
```

Comment:
`TMP` seems to work without also changing `TEMP`.

Reply:
Done.